posix-configs/bebop/mainapp.config

```diff
@@ -4,9 +4,10 @@ param load
 param set SYS_AUTOSTART 4001
 param set MAV_BROADCAST 1
 df_ms5607_wrapper start
+df_mpu6050_wrapper start
+sensors start
 sleep 1
 mavlink start -u 14556 -r 1000000
 sleep 1
-mavlink stream -u 14556 -s ATTITUDE -r 50
-mavlink stream -u 14556 -s ALTITUDE -r 1
+mavlink stream -u 14556 -s HIGHRES_IMU -r 50
```

Comment:
The EKF is not running at the moment and not tested for the Bebop. So, there is no attitude published at the moment. I removed it for now and will add it back, when it is tested.